src/slide-pane/index.ts

```diff
@@ -305,7 +305,14 @@ export class SlidePane extends I18nMixin(ThemedMixin(WidgetBase))<SlidePanePrope
 											},
 											[
 												closeText,
-												w(Icon, { type: 'closeIcon', theme, classes })
+												w(Icon, {
```

Comment:
i'd argue that this sizing should either be a `size` property on the `Icon` widget or simply applied to a wrapping `span` instead. Would be the simplest way I believe.

Reply:
Cool I wrapped the icon in a span and put the styling on that